website/source/docs/config/index.html.md

```diff
@@ -193,6 +193,10 @@ The supported options are:
   * `tls_key_file` (required unless disabled) - The path to the private key
       for the certificate.
       
+  * `tls_min_version` (optional) - If provided, specifies the minimum
+      supported version of TLS. Accepted values are "tls10", "tls11"
+      or "tls12". This defaults to "tls12".
```

Comment:
I think we need a big warning that's like "TLS 1.1 and lower is considered insecure, use at your own risk"